javatests/arcs/showcase/typevariables/VariableTest.kt

```diff
@@ -22,11 +22,11 @@ class VariableTest {
     )
 
     @Test
-    fun shop_redactsSku() = runTest {
+    fun shop_redactsSku() = runBlocking {
         val arc = env.startArc(ShopPlan)
 
         // Ensure that the shop recipe is fully processed.
-        withTimeout(1500) {
+        withTimeout(30000) {
```

Comment:
The only thing I don't like about a withTimeout here is that if the Showcase is meant to be a teaching tool, having `withTimeout` may be a distraction?